### PR TITLE
Drift compensation tracker and HMD requirement

### PR DESCRIPTION
**Pull request by Louka3000**
Doesn't do drift compensation if non-imu head tracker is present

### CODE REVIEW

**Comment by Louka3000** [comment]
Not enough feedback, so I'm keeping the defaults as before for now (off), so this can be merged.

**Review by ButterscotchV — APPROVED** [review]

**Comment by ImUrX** [comment]
fix conflict

**Comment by Louka3000** [comment]
done

File: server/core/src/main/java/dev/slimevr/tracking/trackers/TrackerUtils.kt [hunk 1]
```diff
@@ -55,6 +55,24 @@ object TrackerUtils {
...
+	@JvmStatic
+	fun getNonInternalNonImuTrackerForBodyPosition(
+		allTrackers: List<Tracker>,
+		position: TrackerPosition,
+	): Tracker? = allTrackers.firstOrNull {
+		it.trackerPosition === position &&
+			!it.isImu() &&
+			!it.isInternal &&
+			it.status != TrackerStatus.DISCONNECTED &&
+			it.status != TrackerStatus.ERROR
```
Comment (ImUrX) [inline server/core/src/main/java/dev/slimevr/tracking/trackers/TrackerUtils.kt:73]: maybe `sendData` or `reset` would help here?

Comment (Louka3000) [reply server/core/src/main/java/dev/slimevr/tracking/trackers/TrackerUtils.kt:73]: :+1:

**Review by ImUrX — APPROVED** [review]

**Pull request merged**